sensor/kubernetes/localscanner/cert_refresher.go

```diff
@@ -89,7 +92,8 @@ func ensureCertificatesAreFresh(ctx context.Context, requestCertificates request
 		// send the error to the ticker, so it retries with backoff.
 		return 0, err
 	}
-	log.Infof("successfully refreshed %v", certsDescription)
+	knownServiceTypes := maps.Keys(repository.getKnownServiceTypes())
+	log.Infof("successfully refreshed %v: %+q", certsDescription, knownServiceTypes)
```

Comment:
Is this really correct? What if central would (for whatever reason) send only a subset of the certs for which the types are in `knownServiceTypes`? Those would be persisted but the code here would log that it actually refreshed also the missing ones, no?

Reply:
You are right, I have changed implementation to extract string representation from refreshed certificates